### PR TITLE
Nsb 5.2.24

### PR DESCRIPTION
**Pull request by boblangley**
Updates ServiceControl to NSB 5.2.24

### CODE REVIEW

**Comment by boblangley** [comment]
This PR relies on #1017 and should be applied afterwards. This is only WIP as long as #1017 has not been merged.

File: src/ServiceControl/App.config [hunk 1]
```diff
@@ -14,5 +14,43 @@ These settings are only here so that we can debug ServiceControl while developin
...
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.Data.OData" publicKeyToken="31bf3856ad364e35" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-5.6.4.0" newVersion="5.6.4.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.WindowsAzure.Storage" publicKeyToken="31bf3856ad364e35" culture="neutral" />
+        <bindingRedirect oldVersion="0.0.0.0-4.3.0.0" newVersion="4.3.0.0" />
+      </dependentAssembly>
+    </assemblyBinding>
```
Comment (mikeminutillo) [inline src/ServiceControl/App.config]: I don't know that these changes are necessary as SC has some tricksy code to always load the version on local disk.

File: src/ServiceControl/ServiceControl.csproj [hunk 2]
```diff
@@ -656,7 +655,12 @@
       <Install>false</Install>
     </BootstrapperPackage>
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <ProjectReference Include="..\ServiceControl.LearningTransport\ServiceControl.LearningTransport.csproj">
+      <Project>{12f4b317-b885-4d43-a152-fdb5b0ea2042}</Project>
+      <Name>ServiceControl.LearningTransport</Name>
+    </ProjectReference>
+  </ItemGroup>
```
Comment (mikeminutillo) [inline src/ServiceControl/ServiceControl.csproj]: What's this?

Comment (boblangley) [reply src/ServiceControl/ServiceControl.csproj]: That's not supposed to be in there. I will update the PR when I get home.

File: src/ServiceControl/App.config (hunk 1 above)
Comment (boblangley) [reply src/ServiceControl/App.config]: Ah, OK. I will drop this from the PR.

File: src/ServiceControl/ServiceControl.csproj (hunk 2 above)
Comment (mikeminutillo) [reply src/ServiceControl/ServiceControl.csproj]: ;)

**Pull request merged**